drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-client/src/test/java/org/drools/workbench/screens/scenariosimulation/kogito/client/dmn/KogitoScenarioSimulationBuilderTest.java

```diff
@@ -0,0 +1,117 @@
+        //Fourth column
+        assertEquals(114, modelDescriptor.getFactMappingByIndex(3).getColumnWidth(), 0);
+        assertEquals("INSTANCE 2", modelDescriptor.getFactMappingByIndex(3).getFactAlias());
+        assertEquals(FactIdentifier.EMPTY, modelDescriptor.getFactMappingByIndex(3).getFactIdentifier());
+        assertEquals(ExpressionIdentifier.create("1|2", FactMappingType.EXPECT), modelDescriptor.getFactMappingByIndex(3).getExpressionIdentifier());
+    }
+
+    @Test
+    public void createRULESimulationEmptyFactModuleTuple() {
+        FactModelTuple factMappingTuple = new FactModelTuple(Collections.emptySortedMap(), Collections.emptySortedMap());
+        Simulation simulation = kogitoScenarioSimulationBuilderSpy.createDMNSimulation(factMappingTuple);
+        assertNotNull(simulation);
+        assertEquals(1,simulation.getScenarioWithIndex().size());
+        ScesimModelDescriptor modelDescriptor = simulation.getScesimModelDescriptor();
+        assertNotNull(modelDescriptor);
+        assertEquals(4, modelDescriptor.getFactMappings().size());
+        //First column
+        assertEquals(70, modelDescriptor.getFactMappingByIndex(0).getColumnWidth(), 0);
+        assertEquals(FactIdentifier.INDEX.getName(), modelDescriptor.getFactMappingByIndex(0).getFactAlias());
+        assertEquals(FactIdentifier.INDEX, modelDescriptor.getFactMappingByIndex(0).getFactIdentifier());
+        assertEquals(ExpressionIdentifier.INDEX, modelDescriptor.getFactMappingByIndex(0).getExpressionIdentifier());
+        //Second column
+        assertEquals(300, modelDescriptor.getFactMappingByIndex(1).getColumnWidth(), 0);
+        assertEquals(FactIdentifier.DESCRIPTION.getName(), modelDescriptor.getFactMappingByIndex(1).getFactAlias());
+        assertEquals(FactIdentifier.DESCRIPTION, modelDescriptor.getFactMappingByIndex(1).getFactIdentifier());
+        assertEquals(ExpressionIdentifier.DESCRIPTION, modelDescriptor.getFactMappingByIndex(1).getExpressionIdentifier());
+        //Third column
+        assertEquals(114, modelDescriptor.getFactMappingByIndex(2).getColumnWidth(), 0);
+        assertEquals("INSTANCE 1", modelDescriptor.getFactMappingByIndex(2).getFactAlias());
+        assertEquals(FactIdentifier.EMPTY, modelDescriptor.getFactMappingByIndex(2).getFactIdentifier());
+        assertEquals(ExpressionIdentifier.create("1|1", FactMappingType.GIVEN), modelDescriptor.getFactMappingByIndex(2).getExpressionIdentifier());
+        //Fourth column
+        assertEquals(114, modelDescriptor.getFactMappingByIndex(3).getColumnWidth(), 0);
+        assertEquals("INSTANCE 2", modelDescriptor.getFactMappingByIndex(3).getFactAlias());
+        assertEquals(FactIdentifier.EMPTY, modelDescriptor.getFactMappingByIndex(3).getFactIdentifier());
+        assertEquals(ExpressionIdentifier.create("1|2", FactMappingType.EXPECT), modelDescriptor.getFactMappingByIndex(3).getExpressionIdentifier());
```

Comment:
The same check as above could be stand here:
        assertEquals(FactMappingValueType.NOT_EXPRESSION, modelDescriptor.getFactMappingByIndex(3).getFactMappingValueType());